server/src/main/java/com/google/fhir/gateway/AllowedQueriesChecker.java

```diff
@@ -95,6 +95,11 @@ private boolean requestMatches(RequestDetailsReader requestDetails, AllowedQuery
       return false;
     }
 
+    if (entry.getRequestType() != null
```

Comment:
Is this `if` statement needed? I think the main difference between this and the previous `if` is around case-sensitivity, which I think is minor. So if this is not needed, can you please remove it?

Reply:
Done!